packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -215,16 +215,30 @@
       </div>
       <div class="row">
         <div class="field-wrapper">
-          <a class="btn social-btn" :href="uris.github">
-            Continue With GitHub
-          </a>
+          <input
+            type="button"
+            id="continue-github"
+            value="CONTINUE WITH GITHUB"
+            class="btn social-btn"
+            @click="
+              socialUrl = uris.github;
+              openTermsConditionsDialog();
```

Comment:
I think I imagined this as `@click="socialButtonClickHandler('google')"` or something like that. @francoislehoux-okta any thoughts?

Reply:
Apologies for not listing this suggestion sooner

Reply:
That was my thought yeah, my bad for not being clear enough